Yoast/javascript
# Edit Hungarian verbs list

## Summary


This PR can be summarized in the following changelog entry:

* [yoastseo] edits the list of Hungarian verbs needed for the morphological passive construction by removing the prefixes and leaving in only the stems.
* [Yoast SEO Free] improves the passive voice detection for Hungarian. 

## Relevant technical choices:

*

## Test instructions

This PR can be tested by following these steps:

* Set your site to Hungarian
* Test this functionality in Free
* Open a post and add this passive sentence:
`A nyomozás félrevezetődik.`
* Make sure the passive sentence is recognised
* Remove `félre` from `félrevezetődik`
* The sentence should still be detected correctly

## Impact check

* This PR affects the following parts of the plugin, which may require extra testing:
  *

## UI changes

## Quality assurance


Fixes https://yoast.atlassian.net/browse/LINGO-549
